lib/much-rails/models.rb

```diff
@@ -0,0 +1,7 @@
+module MuchRails; end
+module MuchRails::Models
+end
+
+require "much-rails/models/always_destroyable"
+require "much-rails/models/not_destroyable"
+require "much-rails/models/validate_destroy"
```

Comment:
Ah man, I'm so sorry but we should rename "Models" -> "Records". Can you mass find/replace for me please sir? E.g.

```suggestion
module MuchRails::Records; end

require "much-rails/records/always_destroyable"
require "much-rails/records/not_destroyable"
require "much-rails/records/validate_destroy"
```